src/main.rs

```diff
@@ -1,10 +1,36 @@
-// Copyright 2021 IOTA Stiftung
+// Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
 #![warn(missing_docs)]
 
 //! TODO
 
+use actix::{
+    Actor,
+    System,
+};
+use chronicle::{
+    api::{
+        ChronicleAPI,
+        ShutdownAPI,
+    },
+    config::mongo::{
+        Credential,
+        MongoConfig,
+    },
+};
+
 fn main() {
-    println!("Hello world from Chronicle!");
+    dotenv::dotenv().unwrap();
+    env_logger::init();
+    let system = System::new();
+    let mongo_config =
+        MongoConfig::default().with_credential(Credential::default().with_username("root").with_password("pass"));
+    let res: anyhow::Result<()> = system.block_on(async {
+        let api_addr = ChronicleAPI::new(mongo_config)?.start();
+        tokio::signal::ctrl_c().await.ok();
```

Comment:
why do we ignore these errors?

Reply:
Laziness?

Reply:
:trollface: but can't they be coerced into `anyhow::Error`?